ow_lander/scripts/task_discard_sample.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+from ow_lander import actions
+from ow_lander import node_helper
+from ow_lander import constants
+
+from geometry_msgs.msg import Point
+
+import argparse
+
+parser = argparse.ArgumentParser(
+  formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+  description="Discard sample some height above a position on the terrain.")
+parser.add_argument('--frame', '-f', type=int, default=0,
+  choices=constants.FRAME_ID_MAP.keys(),
+  help="The index of the frame position will be interpreted in "
+       + str(constants.FRAME_ID_MAP))
+parser.add_argument('--relative', '-r', action='store_true', default=False,
+  help="If True, position will be interpreted in the tool's frame")
```

Comment:
Should probably remove `If True`. The bool associated with this command is mostly hidden from the user, except that argparse confuses you by printing `(default: False)`.

Another way to look at it is that `If True` implies that you are expected to type `-r True` instead of just `-r`.

Reply:
Fixed